package.json

```diff
@@ -8,9 +8,9 @@
     "build-production": "yarn run build && cp _redirects ./public/_redirects && cat _production-redirects >> ./public/_redirects",
     "build-staging": "yarn run build && cp _redirects ./public/_redirects && cat _staging-redirects >> ./public/_redirects",
     "build": "gatsby build",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop -p 8001",
```

Comment:
I changed the default port to `80011 because a lot of times I have comply-ui running which defaults to `8000`.